chef/data_bags/crowbar/template-network.json

```diff
@@ -8,7 +8,9 @@
       "enable_tx_offloading": true,
       "mode": "single",
       "teaming": {
-        "mode": 1
+        "mode": 1,
+        "miimon": 100,
+        "xmit_hash_policy": "layer2"
```

Comment:
Do we want both, IE, my current change for global settings plus per-network, or only per-network?